pkgs/development/interpreters/hy/default.nix

```
@@ -1,23 +1,33 @@
-{ stdenv, fetchurl, python2Packages }:
+{ stdenv, python3Packages }:
 
-python2Packages.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "hy";
-  version = "0.17.0";
-
-  src = python2Packages.fetchPypi {
+  version = "0.18.0";
```

usually there's a new line between version and src, otherwise LGTM
```suggestion
  version = "0.18.0";

```